product_docs/docs/biganimal/release/reference/cli/index.mdx

```diff
@@ -50,10 +50,10 @@ chmod +x biganimal
 
 ## Authenticate as a valid user
 
-Before using the CLI to manage BigAnimal, you need to authenticate as a valid BigAnimal user. Use the `create-credential` command to authenticate through the BigAnimal website and assign a refresh token and an access token to a local credential. For example:
+Before using the CLI to manage BigAnimal, you need to authenticate as a valid BigAnimal user. Use the `credential create` command to authenticate through the BigAnimal website and assign a refresh token and an access token to a local credential. For example:
 
 ```shell
-biganimal create-credential \
+biganimal credential create\
   --name "ba-user1" \
   --address "portal.biganimal.com" \
   --port "443"
```

Comment:
need to remove mention of 
 --address "portal.biganimal.com" \
  --port "443"

Reply:
done